contents/blog/internship.md

```diff
@@ -0,0 +1,55 @@
+---
+date: 2024-09-06
+title: Win an internship at PostHog and $25K
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/migrating-hog.png
+featuredImageType: full
+category: Inside PostHog
+---
+
+For the first time _ever_ we are looking to hire 3 interns at PostHog. We very, very rarely hire people with no experience so this is a huge opportunity to join a genuinely world class tech team at a company that is rocketing in growth.
+
+Competition will be fierce, this is how to get a spot.
+
+## Criteria
+
+* You must be a developer but have less than 1 year of professional experience (!)
+* You must be based between GMT+2 to GMT-8
+
+## The competition
+
+Between today and the 30th of September, we are looking for the best new product to be built into PostHog.
+
+You can build _whatever_ you want. We have plenty of public info in terms of what we care about.
+
+We are looking for:
+
+* products we could actually ship to real users
+* things that have an impressive amount of scope
+
+Go nuts - build a CRM, build a customer support, or something else.
+
+## Rules
+
+* We will offer up to the best 3 submissions a (well paid) internship at PostHog and $25K (USD)
```

Comment:
Honestly $25k is ludicrous, already $10k – in addition to the internship – would be amazing

Reply:
i'm surprised (which is great) and felt maybe this was under generous, ie "fuck i need to build an entire product to have a _shot_ at this".

how would you react were you 18 and applying _right now_? I'm really curious.

Reply:
Coming back to this: I think I'd give this a shot if I were about to finish the school year, or have just started summer break. I wouldn't start it in August or September, just because of time constraints. So I think we should get back to this at the end of spring, and give it a serious shot then